Makefile

```diff
@@ -399,9 +399,9 @@ endif
 
 package-docs: build-docs build-docs-capi
 	mkdir -p "package/docs"
-	mkdir -p "package/docs/c/runtime-c-api"
+	mkdir -p "package/docs/c/deprecated/"
 	cp -R target/doc package/docs/crates
-	cp -R lib/c-api/doc/deprecated/html/ package/docs/c/runtime-c-api
+	cp -R lib/c-api/doc/deprecated/html/ package/docs/c/deprecated/
```

Comment:
If we do this, all existing links will be broken. I kept the `runtime-c-api` in the target to have everything working correctly (old links).

Reply:
Reverted :-)